node-js/node-js.md

```diff
@@ -5,10 +5,10 @@
 
 ### Prerequisites
 - JavaScript I - VI lessons
```

Comment:
@yati1998 link this to,
https://github.com/Techtonica/curriculum/tree/master/javascript